benchmark/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenCV Zoo Benchmark
+
+Benchmarking different models in the zoo.
+
+Data for benchmarking will be downloaded and loaded in [data](./data) based on given config.
+
+## Preparation
+
+1. Install `python >= 3.6`.
+2. Install dependencies: `pip install -r requirements.txt`.
+
+## Benchmarking
```

Comment:
can description of how the time is measured.

Reply:
Done.